packages/data-stores/src/plans/types.ts

```diff
@@ -68,6 +68,15 @@ export interface PlanIntroductoryOffer {
 	isOfferComplete: boolean;
 }
 
+export interface PlanCostOverride {
```

Comment:
NIT - I think just `CostOverride` sounds more aligned with the rest (`unpackCostOverrides`, `COST_OVERRIDE_REASONS`, etc.)